packages/excalidraw/change.ts

```diff
@@ -1111,6 +1108,9 @@ export class ElementsChange implements Change<SceneElementsMap> {
         changedElements,
         flags,
       );
+
+      // Need ordered nextElements to avoid z-index binding issues
```

Comment:
Out of curiosity, why is the binding order fir redrawing necessary? (in two or three sentences)

Reply:
Well it's not important for this specific case, but the expectation is that the elementsMap (element array) is ordered by z-index, because in cases when the arrow is dragged, it tests the scene elements map for hovered bindables (getHoveredElementForBinding.